index.d.ts

```diff
@@ -1,9 +1,14 @@
-import { SnakeCase } from "type-fest";
+import { SnakeCase, JsonPrimitive } from "type-fest";
 import { Options as SnakeCaseOptions } from "snake-case";
 
 // eslint-disable-next-line @typescript-eslint/ban-types
 type EmptyTuple = [];
 
+type JsonObject = { [Key in string]: JsonValue };
+type JsonObjectOptional = JsonObject | undefined;
```

Comment:
Type definitions for optional properties such as `DeepObjectType` defined in the type test